packages/x-codemod/README.md

```diff
@@ -212,17 +214,51 @@ The list includes these transformers
 Remove feature flags for stabilized `experimentalFeatures`.
 
 ```diff
- <DataGridPremium
--  experimentalFeatures={{
--    ariaV8: true,
--  }}
- />
+  <DataGridPremium
+-   experimentalFeatures={{
+-     ariaV8: true,
+-   }}
+  />
 ```
 
 ```bash
 npx @mui/x-codemod@latest v8.0.0/data-grid/remove-stabilized-experimentalFeatures <path>
 ```
 
+### Pickers codemods
+
+#### `preset-safe` for pickers v8.0.0
+
+The `preset-safe` codemods for pickers.
+
+```bash
+npx @mui/x-codemod@latest v8.0.0/pickers/preset-safe <path|folder>
```

Comment:
Technically this should be `next` for now on the `master` branch. 🤔 
But in that case, we need to add the comment to not forget to change it. 🙈 

<img width="780" alt="Screenshot 2024-12-03 at 11 12 15" src="https://github.com/user-attachments/assets/ec022d44-7abb-4b1f-aa8a-660640661044">


```suggestion
<!-- #default-branch-switch -->
npx @mui/x-codemod@next v8.0.0/pickers/preset-safe <path|folder>
```

Reply:
Didn't know about the comment. Thanks